APE_specutils.rst

```diff
@@ -0,0 +1,147 @@
+
+A spectroscopy package should provide a representation of the data, tools for reducing spectroscopic observations, analysis tools, and methods for interacting and visualization of the spectra.    In this section, we describe some of the functionality that will be required for a common spectroscopic package.   While equally important, we leave the specification of multi-dimensional spectral objects to future work.  At the same time, we should consider how these tools will be compatible with multi-dimensional data in the future. 
+
+A spectrum describes how the flux changes as a function of the energy.   There are a number of other parameters that can be used to describe a spectrum, and a 1D Spectrum object should have the following properties:
+
+* Dispersion
+* Flux 
+* Errors
+* Units
+* System (ie. heliocentric)
+* meta information about the provenance of the spectra
+
+To ease development, the Spectrum1D object could have energy, wavelength, and frequency properties that convert the dispersion to the appropriate dimensions.  In addition to this information, software should provide a method for reading and writing formats regularly used for spectroscopy.   Common formats include iraf, SDSS, *please list others*, midas, FITS tables, and ascii.   Reading and writing into a 1D spectrum object should use the astropy io register.   In addition, methods should exist for arithmetic on spectra1d objects including how to handle objects with different dispersion sampling.  
+
+Spectrum1D will define a specific interpretation of the meaning of a pixel coordinate value (i.e., is it the mean? Left side? Median? In what units).  This sentence is a placeholder; we plan to have a breakout session on April 21 to identify whether there is a standard we can select and finalize this choice.
+
+
+Specifications of Other Things to be included in specutils
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Because of the diverse set of analysis requirements and reduction techniques for astronomical spectroscopy, we do not focus on specific requirements for the reduction or analysis of spectra, but the aspects that are similar regardless of the type of spectroscopy. This functionality will be included in `specutils` so that it does not need to be duplicated in various domain-specific tools.   Some of the common functionality that is required for spectroscopic packages include:
+
+* Identification of spectral lines or other features
+* Measuring features in the spectra including fitting models, calculating line centroids/shapes, and fluxes
+* Transforming the spectra to different unit systems (e.g., converting between different velocity frames, frequency vs wavelength, flux transformations)
+* Measuring velocities for the spectra including cross-correlation
+* deblending of lines
+* Interpolation, either onto different wavelengths (with flux-conserving at least an option) or over bad data points
+* Continuum subtraction or normalization
+
```

Comment:
Just a heads-up, this proved to be a topic of disagreement/difficulty in the past, since dealing with errors is not trivial etc. Not saying it can't be done, but probably shouldn't be in the base class.